server/views/partials/prisonerBanner.njk

```diff
@@ -1,17 +1,23 @@
-<div class="app-prisoner-banner govuk-!-display-none-print">
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-visually-hidden">{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s details</p>
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">Prison number: <span class="govuk-body" data-qa="prison-number">{{ prisonerSummary.prisonNumber }}</span></p>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">Entered this prison on: <span class="govuk-body" data-qa="reception-date">{{ prisonerSummary.receptionDate | formatDate('D MMM YYYY') }}</span></p>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">Release date: <span class="govuk-body" data-qa="release-date">{{ prisonerSummary.releaseDate | formatDate('D MMM YYYY') or 'N/A' }}</span></p>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-6">DOB: <span class="govuk-body" data-qa="dob">{{ prisonerSummary.dateOfBirth | formatDate('D MMM YYYY') }}</span></p>
-    </div>
-  </div>
+<div class="app-mini-profile-header govuk-!-margin-bottom-6">
+  <dl>
+    <dt class="govuk-heading-s govuk-!-margin-bottom-0">
+      <a href="{{ dpsUrl }}/prisoner/{{ prisonNumber }}" class="govuk-link govuk-!-font-weight-bold">
```

Comment:
I'm assuming this is the page we want to link to; the ticket nor the prototype had the link.

Reply:
That seems like a safe assumption to me, but make sure you check with Aliki or someone, or specifically call it out for them to test when we pass it over to the team to test